osbs/cli/main.py

```diff
@@ -497,12 +497,20 @@ def cmd_backup(args, osbs):
     else:
         outfile = dirname + ".tar.bz2"
 
-    with paused_builds(osbs, quota_name='pause-backup'):
+    with paused_builds(osbs, quota_name='pause-backup',
+                       ignore_quota_errors=args.ignore_quota_errors):
         with TarWriter(outfile, dirname) as t:
             for resource_type in BACKUP_RESOURCES:
-                logger.info("dumping %s", resource_type)
-                resources = osbs.dump_resource(resource_type)
-                t.write_file(resource_type + ".json", json.dumps(resources).encode('ascii'))
+                try:
+                    logger.info("dumping %s", resource_type)
+                    resources = osbs.dump_resource(resource_type)
+                    t.write_file(resource_type + ".json", json.dumps(resources).encode('ascii'))
+                except Exception as e:
```

Comment:
I think not - a lot of different errors may occur - timeouts, missing permissions, write errors etc. Not sure if there is a more specific base class which would catch all of these